docs/api/collection.md

```diff
@@ -55,6 +59,9 @@ Fetch a specific Collection's items with the following options:
                    when `is_not_pinned`, return non pinned objects only.
                    when `all`, return everything. By default returns everything.
 
+  Note that this endpoint should return results in a similar shape to `/api/dashboard/:id/items`, so if this is
```

Comment:
This seems like an internal note that should be a code comment rather than a docstring